test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/otel-collector-config/otel-collector.crt

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
One concern... this cert will expire in a year (`Valid To : Feb 05,2022`) and our tests will break when that happens. We might be able to have the docker compose automatically generate one on the fly so that it will always be valid. Google turned this up: https://stackoverflow.com/questions/44047315/generate-a-self-signed-certificate-in-docker

Also having a cert in the repo might flag us on some security scans. We know it is benign but it could be a headache 😉

Reply:
If it was ok with everyone else, I was just going to set up an automated email reminder for @CodeBlanch to renew the cert each year 😆. No, kidding... I had actually created this cert a number of times trying to get it right and I had meant to do it for 10 years.

But, your thought is waaaay better - just create a new cert for each test run. I'll need to play around with this because the cert is needed by both the collector container and the container the test is run in, but pretty sure I can figure this out.

Reply:
Just checked out the latest changes, very cool solution!